default.nix

```diff
@@ -189,7 +189,7 @@ let
       client = let
         generated-purescript = pkgs.runCommand "plutus-playground-purescript" {} ''
           mkdir $out
-          ${haskellPackages.plutus-playground-server}/bin/plutus-playground-server psgenerator $out
+          ${playground-exe}/bin/plutus-playground-server psgenerator $out
```

Comment:
So we'll still rebuild the client, I guess?

Reply:
ah good point, I can use the original haskellPackage here

Reply:
I thought the point was that we couldn't, because it writes it out as part of the generation?

Reply:
agh, indeed and I don't think there's a similar way to do this in purescript at the moment unless you wrote your own thing to hack into the compiled javascript (which you could). I'm going to suggest let's not bother doing that right now.